docs/API.md

```diff
@@ -444,7 +446,7 @@ Options can be configured as a parameter to the [init()](./interfaces/tracer.htm
 | profiling       | `DD_PROFILING_ENABLED`             | `false`        | Whether to enable profiling. |
 | reportHostname  | `DD_TRACE_REPORT_HOSTNAME`         | `false`        | Whether to report the system's hostname for each trace. When disabled, the hostname of the agent will be used instead. |
 | ingestion.sampleRate | `DD_TRACE_SAMPLE_RATE`        | `-`            | Controls the ingestion sample rate (between 0 and 1) between the agent and the backend. Defaults to deferring the decision to the agent. |
-| ingestion.rateLimit  | `DD_TRACE_RATE_LIMIT`         | `-`            | Controls the ingestion rate limit between the agent and the backend. Defaults to deferring the decision to the agent. |
+| ingestion.rateLimit | `DD_TRACE_RATE_LIMIT`          | `-`            | Controls the ingestion rate limit between the agent and the backend. Defaults to deferring the decision to the agent. |
```

Comment:
Let's not document this since it should no longer be used and is effectively deprecated.

Reply:
Same for the sample rate above.

Reply:
Should the be removed from `index.d.ts` as well?

Reply:
The idea is that it should still work, but it should be deprecated and shouldn't be documented. So I would:

- Remove from the documentation entirely.
- Mark as deprecated in the TypeScript definitions.
- Keep in code so that it's backward compatible.